src/custom-webpack/dev-server/index.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by Evgeny Barabanov on 28/06/2018.
+ */
+
+import {BuilderContext} from '@angular-devkit/architect';
+import {CustomWebpackDevServerBuildSchema} from './schema';
+import {DevServerBuilder, NormalizedBrowserBuilderSchema} from '@angular-devkit/build-angular';
+import {getSystemPath, Path, virtualFs} from '@angular-devkit/core';
+import * as fs from 'fs';
+
+const webpackMerge = require('webpack-merge');
+
+export class CustomWebpackDevServerBuilder extends DevServerBuilder {
+
+  constructor(public context: BuilderContext) {
+    super(context);
+  }
+  buildWebpackConfig(root: Path,
+                     projectRoot: Path,
+                     host: virtualFs.Host<fs.Stats>,
+                     options: NormalizedBrowserBuilderSchema) {
```

Comment:
Should be DevServerBuilderOptions instead of NormalizedBrowserBuilderSchema

Reply:
Shoud be `BrowserBuilderSchema` right?

The original file is it
https://github.com/angular/angular-cli/blob/44086c60ff1d6c26d30850bef125120f6c498ac1/packages/angular_devkit/build_angular/src/dev-server/index.ts#L178

Reply:
Yep you're correct